### PR TITLE
chore(docs):  automatically add `rel` and `target` attrs to external links + link formatting

### PR DESCRIPTION
**Pull request by tmorehouse**
### Describe the PR

Note: only converts links generated from markdown formatted links (i.e. `[text](url)`)

### PR checklist

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [x] Enhancement
- [ ] ARIA accessibility
- [ ] Documentation update
- [ ] Other (please describe)

**Does this PR introduce a breaking change?** (check one)

- [x] No
- [ ] Yes (please describe)

**The PR fulfills these requirements:**

- [x] It's submitted to the `dev` branch, **not** the `master` branch
- [ ] When resolving a specific issue, it's referenced in the PR's title (i.e. `[...] (fixes #xxx[,#xxx])`, where "xxx" is the issue number)
- [x] It should address only one issue or feature. If adding multiple features or fixing a bug and adding a new feature, break them into separate PRs if at all possible.
- [x] The title should follow the [**Conventional Commits**](https://www.conventionalcommits.org/) naming convention (i.e. `fix(alert): not alerting during SSR render`, `docs(badge): update pill examples, fix typos`, `chore: fix typo in README`, etc). **This is very important, as the `CHANGELOG` is generated from these messages.**

**If new features/enhancement/fixes are added or changed:**

- [ ] Includes documentation updates (including updating the component's `package.json` for slot and event changes)
- [ ] New/updated tests are included and passing (if required)
- [ ] Existing test suites are passing
- [ ] The changes have not impacted the functionality of other components or directives
- [ ] ARIA Accessibility has been taken into consideration (Does it affect screen reader users or keyboard only users? Clickable items should be in the tab index, etc.)

**If adding a new feature, or changing the functionality of an existing feature, the PR's
description above includes:**

- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

### CODE REVIEW

**Comment by jackmu95** [comment]
@tmorehouse While at it we should take a look at all `.md` files and convert HTML anchors to Markdown syntax if possible.

**Comment by tmorehouse** [comment]
@jackmu95 I think I've got all the links converted.  Also added in `rel="noopener"` to all external links

File: docs/nuxt.config.js [hunk 1]
```diff
@@ -14,6 +14,10 @@ hljs.registerLanguage('bash', require('highlight.js/lib/languages/bash')) // inc
 hljs.registerLanguage('shell', require('highlight.js/lib/languages/shell'))
 hljs.registerLanguage('plaintext', require('highlight.js/lib/languages/plaintext'))
 
+// Constants which should be grabbed from package.json
+// Bootstrap minor version (for links)
+const bootstrapDocsVersion = '4.3'
```
Comment (jackmu95) [inline docs/nuxt.config.js]: The more I think about it I don't like this change...
Maybe it's better to just run a search and replace on the `.md` files.

The other changes are good!

Comment (tmorehouse) [reply docs/nuxt.config.js]: Yeah, it might be easy for future devs to forget to update this variable.

Comment (tmorehouse) [reply docs/nuxt.config.js]: Removed that part of the PR

**Pull request merged**